src/index.d.ts

```diff
@@ -0,0 +1,6 @@
+export const alphabetize: (sentenceArray: string[], type: 'english' | 'unicode') =>  string;
+export const  ascii: (letters: string) => string ;
+export const firstLetters: (words: string, eng?:boolean, simplify?: boolean) => string;
+export const mainLetters: (words: string, simplify?: boolean, simplifyConsonants?: boolean) =>  string;
+export const translit: (gurmukhi: string) =>  string;
+export const unicode: (text: string, reverse?: boolean, simplify?: boolean) => string;
```

Comment:
text is defaulted too https://github.com/KhalisFoundation/anvaad-js/blob/master/src/unicode.js#L373

This is fine though. No use case I can see where a typescript user would call this function with no params.